HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -37,7 +37,10 @@
         "STORAGE_CONTAINER_NAME": "housing-repairs-online",
         "GOV_NOTIFY_KEY" : "<REPLACE WITH GOV NOTIFY API URL>",
         "CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY SMS TEMPLATE ID>",
-        "CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY EMAIL TEMPLATE ID>"
+        "CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY EMAIL TEMPLATE ID>",
```

Comment:
we might want to save the template ID's in here too since we're saving the `INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID`